python/setup.py.in

```diff
@@ -24,7 +24,7 @@ if '${CMAKE_SYSTEM_PROCESSOR}' not in ['arm', 'armv7-a', 'aarch64']:
     setup_requires+=["opencv-python"]
 
 # the prefix is sys.prefix which should always be usr
-paddle_bin_dir = 'local/opt/paddle/bin'
+paddle_bin_dir = 'opt/paddle/bin'
```

Comment:
Thanks @tensor-tang , but maybe it doesn't matter for the different path? 
If users want to install paddle to a customer directory, he/she could set `--prefix=<customer directory>`, but in the production Docker image, we just make it running correctly.

BTW, the directory in `date_files` wouldn't be configured `local/opt/paddle/bin`, this will caused paddle be installed under `/usr/local/local/opt/paddle/bin`